R/uniDimBayesianCalcCoeff.R

```diff
@@ -98,14 +109,18 @@
 
     if (is.null(out[["samp"]])) {
       startProgressbar(model[["progressbarLength"]])
-      out[["samp"]] <- coda::mcmc(apply(model[["gibbsSamp"]], MARGIN = c(1, 2), Bayesrel:::applyalpha, progressbarTick))
+
+      out[["samp"]] <- coda::mcmc(apply(model[[if (options[["stdCoeffs"]] == "unstand") "gibbsSamp" else "gibbsCor"]],
+                                        MARGIN = c(1, 2), Bayesrel:::applyalpha, progressbarTick))
```

Comment:
well but would I then need a special function that just does that? Because I dont want any of the other functions to depend on the standardized button at all. 
something like : 
```r
model[["type"]] <- function(options) {
return(if (options[["stdCoeffs"]] == "unstand") "gibbsSamp" else "gibbsCor")
}
```

Reply:
Ah, never mind, these are all independent functions, I missed that.
This solution is fine as is.